Lib/urllib/parse.py

```diff
@@ -363,7 +408,7 @@ def _fix_result_transcoding():
 _fix_result_transcoding()
 del _fix_result_transcoding
 
-def urlparse(url, scheme='', allow_fragments=True):
+def urlparse(url, scheme='', allow_fragments=True, flags=SchemeFlag(0)):
```

Comment:
what does the `0` mean?

Reply:
@gpshead Somewhere I thought you said it should be `flags=None` instead -- I agree.

Reply:
None is used - `677ed1aac3`.